setup.cfg

```diff
@@ -1,5 +1,13 @@
+[bumpversion]
+current_version = 0.0.1
```

Comment:
0.1.0, would be when features are solidified and ready for the first beta IMO.

I would start at 0.0.1 for now.